### PR TITLE
[USM] Support for postgres truncate operation

### PR DESCRIPTION
**Pull request by amitslavin**
### What does this PR do?

Adds support for TRUNCATE operation in Postgres.

### Motivation

Extend our Postgres support capabilities and increase our accuracy.

### Additional Notes

TODO:

- [x] Merge the agent payload [PR](https://github.com/DataDog/agent-payload/pull/316) and update the `go.mod`

### CODE REVIEW

File: pkg/network/usm/postgres_monitor_test.go
Comment (guyarb) [inline pkg/network/usm/postgres_monitor_test.go:1]: what about classification test?

File: go.mod [hunk 1]
```diff
@@ -601,7 +601,7 @@ require github.com/lorenzosaino/go-sysctl v0.3.1
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
-	github.com/DataDog/agent-payload/v5 v5.0.118
+	github.com/DataDog/agent-payload/v5 v5.0.119-0.20240527120349-da3a257b7af6
```
Comment (guyarb) [inline go.mod:604]: change to a formal version
don't forget to open a matching PR in dd-go

**Review by hmahmood — APPROVED** [review]

**Review by guyarb — APPROVED** [review]

**Comment by guyarb** [comment]
CI fails

**Comment by amitslavin** [comment]
@guyarb, It seems like the CI is fine, probably because the flaky test is now being skipped.

**Pull request merged**